Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM denoland/deno:alpine-1.37.1
+FROM denoland/deno:alpine-1.38.3
 
+RUN mkdir /etc/connector
 COPY ./src /app
-COPY ./functions /functions 
-WORKDIR /functions
-
-RUN ls /app
+COPY ./functions /functions/src
+WORKDIR /functions/src
 
 # Pre-cache inference results and dependencies
-RUN EARLY_ENTRYPOINT_EXIT=true sh /app/entrypoint.sh
+RUN PRECACHE_ONLY=true /app/entrypoint.sh
```

Comment:
We should probably just split the initial actions into a seperate script.

Reply:
What, and call it from this script? It still needs to happen on container start for those volume-mounting their functions.

Reply:
Call the build actions from the dockerfile directly or via a build script, then remove them from the entrypoint script

Reply:
Oh right... yeah this would work on connector create, but not for people doing local docker... bummer